composables/helpers/normalizeCart.ts

```diff
@@ -0,0 +1,21 @@
+import { Cart } from "@/server/types/GraphQL"
+
+export const normalizeCart = (cart: Cart) => {
```

Comment:
Add a utitliy function that handles mapping "DirectShip" and "Pickup" and use that where-ever its needed. Don't create a new normalizeCart function just to remap these

Reply:
I am removing this as of now because we are not using it right away. Should be a separate ticket for this. Need to change a lot of code that is based upon the nuxt config fulfillment options.